Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -224,6 +211,23 @@ extension AlertManager {
 // MARK: Extensions
 
 extension FileManager {
+    
+    fileprivate func ensureDirectoryExists(_ directory: URL?, log: DiagnosticLog) {
```

Comment:
[nit] maybe return success/failure, so caller can decide what to do in those cases?